glib/src/gobject/type_info.rs

```diff
@@ -0,0 +1,6 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[derive(Debug)]
+#[doc(alias = "GTypeInfo")]
+#[repr(transparent)]
+pub struct TypeInfo(pub gobject_ffi::GTypeInfo);
```

Comment:
```suggestion
pub struct TypeInfo(pub(crate) gobject_ffi::GTypeInfo);
```

Maybe, plus an `as_ptr()` function that returns the underlying FFI pointer?